app/jobs/data_exports/create_job.rb

```diff
@@ -30,11 +30,12 @@ def initialize_manifest(export_type)
     def create_sample_zip(data_export)
       new_zip_file = Tempfile.new(binmode: true)
       ZipKit::Streamer.open(new_zip_file) do |zip|
-        data_export.export_parameters['ids'].each do |sample_id|
-          sample = Sample.find(sample_id)
+        samples = Sample.includes(attachments: { file_attachment: :blob })
```

Comment:
**nitpick (blocking):** if you add `:project, ` inside the `includes` line 38 shouldn't cause another query to get the project.

Reply:
Fixed in fe2ede75f9d4240f8b71eefb3d4dce58624b27ed